source/ada/lsp-lal_utils.ads

```diff
@@ -43,6 +46,21 @@ package LSP.Lal_Utils is
    --  files from the disk.
    --  If Include_Definition is True, include the definition as well.
 
+   --------------------------
+   -- Document abstraction --
```

Comment:
We already have document abstraction in LSP_Documents. I doubt we need another one.